infrastructure/template.yaml

```diff
@@ -734,7 +734,8 @@ Outputs:
 
   PersonalIdenityTable:
     Description: "Get Personal Identity Name"
-    Value: !Sub "{{resolve:ssm:/${CommonStackName}/PersonIdentityTableName}}"
+    # Value: !Sub "{{resolve:ssm:/${CommonStackName}/PersonIdentityTableName}}"
+    Value: !Ref PersonalIdentityTable
```

Comment:
For now this is OK - we will need to revert this once we have the nino in the real person identity table

Reply:
agree